### ISSUE TITLE
why use log in this loss function

### ISSUE THREAD

**Issue by HuinanZ** [Mamiglia/WeightedKappaLoss#1]
When using log, the weightedkappaloss is in ($-\infty$, log2), which will be negative. In torch training, loss which is negative may have a bad effect on the period. Removing the log will influence the training process or not? Maybe I cannot understand the mathematical processes well. Thank u for your help!
